src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -48,7 +48,7 @@ This is a checklist for web.dev content reviews. It includes critical high-level
 1. Are all links to web.dev pages and assets relative?
```

Comment:
Link or explanation to what it means to be relative?

Reply:
Will link to this: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#Examples_of_relative_URLs